.github/workflows/client_workflow.yml

```diff
@@ -1,6 +1,9 @@
 name: build_client
 
-on: [push, pull_request]
+on: 
+  pull_request:
```

Comment:
What is the reason behind removing the GitHub action on push? Wouldn't be better to have it on push too in order to fix whatever problem we might encounter?

Reply:
yes i agree

Reply:
So right now, with the new configuration, it will only run on pushes to a Pull Request, and when the Pull Request is opened. Previously, it would run on that and on every push to github as well. What do you guys prefer?